lib/voice/VoiceConnection.js

```diff
@@ -22,7 +22,8 @@ try {
 let Sodium = null;
 let NaCl = null;
 
-const ENCRYPTION_MODE = "xsalsa20_poly1305";
+const VOICE_GATEWAY_VERSION = 8;
```

Comment:
this might be worth moving into `lib/Constants.js` as `Constants.VOICE_VERSION` or something like that like is currently with REST and the regular gateway